robottelo/ui/computeresource.py

```diff
@@ -137,10 +135,107 @@ def update(self, name, newname=None, parameter_list=None,
         self.click(common_locators['submit'])
 
     def delete(self, name, really=True):
-        """Removes the compute resource entity"""
+        """Removes the compute resource entity."""
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def access_compute_resource(self, res_name):
+        """Access a particular compute resource."""
+        strategy, value = locators['resource.get_by_name']
+        resource = self.search(res_name)
+        if resource is None:
+            raise UINoSuchElementError(
+                'Could not find the resource {0}'.format(res_name))
+        self.click((strategy, value % res_name))
```

Comment:
Now you can use locators directly as: 
`self.click(locators['resource.get_by_name'] % res_name)`

Not a blocker, just a comment (in future we should change every occurrence)